tools/rvn/CommandLineApp.cs

```diff
@@ -575,6 +653,11 @@ private static CommandOption ConfigureCertPassword(CommandLineApplication cmd)
             return cmd.Option("-p|--password", $"Certificate password{Environment.NewLine}Password can be set from ENV:{Environment.NewLine}Windows - $env:RVN_CERT_PASS=password\nLinux - export RVN_CERT_PASS=password", CommandOptionType.SingleValue);
         }
 
+        private static CommandOption ConfigureGenerateValues(CommandLineApplication cmd)
+        {
+            return cmd.Option("--generate-helm-values", "Path to values.yaml", CommandOptionType.SingleOrNoValue);
```

Comment:
Doesn't this method have a parameter for supplying default value?
Or a property after it's created?